packages/common/random-access-storage/src/browser/storage.test.ts

```diff
@@ -2,12 +2,10 @@
 // Copyright 2021 DXOS.org
 //
 
-// @dxos/test platform=browser
-
-import expect from 'expect';
 import 'source-map-support/register';
 
-import { describe, test } from '@dxos/test';
+import { expect } from 'chai';
```

Comment:
I think vitest provides expect as well https://vitest.dev/guide/features.html#chai-and-jest-expect-compatibility probably better to start migrating to that to simplify dependencies

Reply:
makes sense, I left it cause I though maybe the team preferred chai syntax, changed to expect from vitest